### ISSUE TITLE
https://huggingface.co/Mrkomiljon/voiceGUARD us returning 7 Classes

### ISSUE THREAD

**Issue by ibliskavka** [Mrkomiljon/voiceguard#1]
Hello!

I am new to ML but I am very interested in the voiceGUARD model you have on HuggingFace.

I attempted to run the model and noticed that the `logits` contains an array of 7 values. According to the docs it should only return 2. What are the other classes?

Any help would be appreciated.
